app/src/main/java/com/randomappsinc/studentpicker/settings/SettingsFragment.java

```diff
@@ -122,6 +126,14 @@ public void onItemClick(int position) {
             case 6:
                 intent = new Intent(Intent.ACTION_VIEW, Uri.parse(REPO_URL));
                 break;
+
+            case 7:
+                PremiumFeatureOpener.openFeature(R.string.restore_name_list_from_backup, requireActivity(), () -> {
```

Comment:
Adding some context here: This feature is at the bottom of the settings list in Simple Flashcards Plus, because that app has full-stack accounts. Given that this app is local, this feature should be after dark mode IMHO.
```suggestion
            case 7:
                PremiumFeatureOpener.openFeature(R.string.restore_name_list_from_backup, requireActivity(), () -> {
```